torchrl/envs/transforms/transforms.py

```diff
@@ -2489,8 +2489,10 @@ def __init__(
                 )
         else:
             in_keys = sorted(in_keys, key=_sort_keys)
-        if type(out_key) != str:
-            raise Exception("CatTensors requires out_key to be of type string")
+        if not isinstance(out_key, str) and not (
+            isinstance(out_key, tuple) and set(map(type, out_key)) == {str}
+        ):
```

Comment:
See https://github.com/pytorch-labs/tensordict/pull/479/files#r1254257836

I wonder if this is needed

Reply:
should we just chek it is not a list?